src/Uno.UI/UI/Xaml/Media/PointCollection.cs

```diff
@@ -10,30 +10,102 @@ namespace Windows.UI.Xaml.Media
 {
 	public partial class PointCollection : IEnumerable<Windows.Foundation.Point>, IList<Windows.Foundation.Point>
 	{
-		private List<Windows.Foundation.Point> _coords;
+		private readonly List<Windows.Foundation.Point> _points;
+		private readonly List<Action> _changedCallbacks = new List<Action>();
 
 		public PointCollection()
 		{
-			_coords = new List<Foundation.Point>();
+			_points = new List<Foundation.Point>();
 		}
 
 		public PointCollection(IEnumerable<Windows.Foundation.Point> coordinates)
 		{
-			_coords = coordinates.ToList();
+			_points = coordinates.ToList();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'coordinates' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5289016)